tensornetwork/contractors/custom_path_solvers/examples/example.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 The TensorNetwork Authors
```

Comment:
Can you move this file to `examples/custom_path_solvers`?

Reply:
No problem. Just to be sure, you mean move this to a folder: TensorNetwork/examples/custom_path_solvers

Reply:
no just `examples/custom_path_solvers`.

This directory
https://github.com/google/TensorNetwork/tree/master/examples